templates/individual-sponsors.html

```diff
@@ -46,8 +46,16 @@ <h3 class="sponsors__amount">${{ tier.amount }} / month</h3>
         {% for sponsor in sponsors_in_tier %}
         {% if tier.reward_logo %}
         <a class="sponsors__link" href="{{ sponsor.link }}">
-            <img class="sponsors__logo" {% if sponsor.style %}style="{{ sponsor.style }}"{% endif %} src="/assets/sponsors/{{ sponsor.image }}"
-                alt="{{ sponsor.name }} logo" />
+            <img
+                class="sponsors__logo"
+                style="height:
```

Comment:
I disagree with using inline styling for setting the height for an `<img />`. This should probably instead use the template language to calculate a value for the height attribute in a way like:

NOTE: I'm not sure the exact syntax that Tera uses for variable calculations, so this may be slightly off.
```
<img src="..." height="
  {%- if sponsor.square_logo and sponsor.logo_scale -%}
    {{ tier.logo_height * 1.666 * sponsor.logo_scale }}
  {%- elif sponsor.square_logo and not sponsor.logo_scale -%}
    {{ tier.logo_height * 1.666 }}
  {%- elif sponsor.logo_scale and not sponsor.square_logo -%}
    {{ tier.logo_height * tier.logo_scale }}
  {%- endif -%} />
```

Reply:
I'm assuming you mean that you disagree with using css `calc()`, as your suggestion also uses "inline styling". Assuming that, I agree that pre-calculating makes sense.

Reply:
My solution isn't using "inline styling" for inline styling is, for all intents and purposes, non-semantic CSS styling in the `style` attribute. This is instead using the semantic height attribute of the image tag that has more meaning to the HTML and rendering.

Reply:
Ah yeah I missed the part of your suggestion where you set the `height` attribute directly on the `img`. Just pushed similar changes.

I rephrased your suggested logic to be less combinatorically nasty when it comes to boolean logic.

I also kept the optional style override as a break-glass solution if we need to tweak a donor logo in a non-standard way.